packages/salesforcedx-apex-replay-debugger/test/unit/adapter/apexReplayDebug.test.ts

```diff
@@ -62,6 +63,15 @@ export class MockApexReplayDebug extends ApexReplayDebug {
   public shouldStopForBreakpoint(): boolean {
     return super.shouldStopForBreakpoint();
   }
+
+  public setProjectPath(projectPath: string | undefined): void {
+    this.projectPath = projectPath;
+  }
+
+  public getProjectPath(): string | undefined {
+    const foo = this.projectPath;
```

Comment:
This has already been changed locally to just return this.projectPath and will be updated with the next push that'll be based on feedback. This was a leftover tweak from when I'd had to debug something.